runtime/compiler/spotbugs.vim

```diff
@@ -14,22 +14,74 @@ set cpo&vim
 " The regex, auxpath and glob try to include all dependent classes of the
 " current buffer. See https://github.com/spotbugs/spotbugs/issues/856
 
+" FIXME: When "search()" is used with the "e" flag, it makes no _further_
+" progress after claiming an EOL match (i.e. "\_" or "\n", but not "$").
+" XXX: Omit anonymous class declarations
+let s:types = '\<\%(\.\@1<!class\|@\=interface\|enum\|record\)\%(\s\|$\)'
+let s:names = '\<\%(\.\@1<!class\|@\=interface\|enum\|record\)\s*\(\K\k*\)\>'
```

Comment:
According to `:help match-pattern`,

https://github.com/vim/vim/blob/7724d621f21ff6599ba49f11401ddb45ff321099/runtime/doc/builtin.txt#L6894-L6898

and `:help search()`,

https://github.com/vim/vim/blob/7724d621f21ff6599ba49f11401ddb45ff321099/runtime/doc/builtin.txt#L8971

we may want to consider prepending `\C` to both patterns.